test/builtin_dataset_mocks.py

```diff
@@ -911,6 +911,61 @@ def country211(info, root, config):
     return num_examples * len(classes)
 
 
+@register_mock
+def food101(info, root, config):
+    data_folder = root / "food101"
```

Comment:
Please re-check the original data, but I think we need this dash here.

```suggestion
    data_folder = root / "food-101"
```

Reply:
At first, I have also used the "food-101" name but then it was a hassle for naming the files and doing the imports in the code. Should I keep this name everywhere I deal with the imports with the "-" on them or is it only needed in the tests?

Reply:
Got it. I thought this had an overall impact. Thanks for the clarification.